apps/backend/package.json

```diff
@@ -8,6 +8,7 @@
     "start": "ts-node -r tsconfig-paths/register src/index.ts",
     "build": "prisma generate && tsc --project tsconfig.json && tsc-alias",
     "serve": "pnpm ts-node -r tsconfig-paths/register dist/apps/backend/src/index.js",
+    "type-check": "tsc",
```

Comment:
Added typing check command to both frontend and backend. I'll probably add some kind of pre-commit hook next to prevent build failures.